astro/src/content/docs/_shared/_userinfo-response.mdx

```diff
@@ -30,72 +24,42 @@ import InlineField from 'src/components/InlineField.astro';
   </APIField>
-  <APIField name="name" type="String" since="1.1.0" renderif={props.show_since}>
+  <APIField name="name" type="String" since="1.1.0" show_since={props.show_since}>
     The full name of the User if available.
 
     In version `1.50.0` and later, when the <InlineField>Scope handling policy</InlineField> is `Strict`, this field is only populated when the provided token contains the `profile` scope.
   </APIField>
-  <APIField name="name" type="String" renderif={!props.show_since}>
-    The full name of the User if available.
-
-    In version `1.50.0` and later, when the <InlineField>Scope handling policy</InlineField> is `Strict`, this field is only populated when the provided token contains the `profile` scope.
-  </APIField>
-  <APIField name="middle_name" type="String" since="1.1.0" renderif={props.show_since}>
-    The middle name of the User if available.
-
-    In version `1.50.0` and later, when the <InlineField>Scope handling policy</InlineField> is `Strict`, this field is only populated when the provided token contains the `profile` scope.
-  </APIField>
-  <APIField name="middle_name" type="String" renderif={!props.show_since}>
+  <APIField name="middle_name" type="String" since="1.1.0" show_since={props.show_since}>
     The middle name of the User if available.
 
     In version `1.50.0` and later, when the <InlineField>Scope handling policy</InlineField> is `Strict`, this field is only populated when the provided token contains the `profile` scope.
   </APIField>
-  <APIField name="phone_number" type="String" since="1.1.0" renderif={props.show_since}>
-    The phone number of the User if available.
+  <APIField name="phone_number" type="String" since="1.50.0" show_since={props.show_since}>
+    If available, the phone number of the User (<InlineField>user.phoneNumber</InlineField> or <InlineField>user.mobilePhone</InlineField> will be used in order of precedence).
 
-    In version `1.50.0` and later, when the <InlineField>Scope handling policy</InlineField> is `Strict`, this field is only populated when the provided token contains the `phone` scope.
+    When the <InlineField>Scope handling policy</InlineField> is `Strict`, this field is only populated when the provided token contains the `phone` scope.
   </APIField>
-  <APIField name="phone_number" type="String" renderif={!props.show_since}>
-    The phone number of the User if available.
+  <APIField name="phone_number_verified" type="Boolean" since="1.99.9" show_since={props.show_since}>
+    The OpenId Connect claim indicating if the User's phone number has been verified.
 
-    In version `1.50.0` and later, when the <InlineField>Scope handling policy</InlineField> is `Strict`, this field is only populated when the provided token contains the `phone` scope.
+    When the `Scope handling policy` is `Strict`, this field is only populated when the <InlineField>scope</InlineField> claim contains the `phone` scope.
```

Comment:
I think the decision was to not populate this field in `Compatibility` mode. If so, the comment should make it clearer that this is only populated when both conditions are true.
```suggestion
    This field is only populated when the `Scope handling policy` is `Strict`, and the <InlineField>scope</InlineField> claim contains the `phone` scope.
```

Reply:
Fixed, got loose with cut and pasting that line.